src/datasets/arrow_dataset.py

```diff
@@ -344,32 +344,40 @@ def to_tf_dataset(
-        the Dataset, and is suitable for passing to methods like model.fit() or model.predict(). The dataset will yield
-        dicts for both inputs and labels unless the dict would contain only a single key, in which case a raw
-        tf.Tensor is yielded instead.
+        """Create a `tf.data.Dataset` from the underlying Dataset. This `tf.data.Dataset` will load and collate batches from
+        the Dataset, and is suitable for passing to methods like `model.fit()` or `model.predict()`. The dataset will yield
+        `dicts` for both inputs and labels unless the `dict` would contain only a single key, in which case a raw
+        `tf.Tensor` is yielded instead.
 
         Args:
-            batch_size (:obj:`int`): Size of batches to load from the dataset.
-            columns (:obj:`List[str]` or :obj:`str`, optional): Dataset column(s) to load in the tf.data.Dataset. Column
-             names that are created by the `collate_fn` and that do not exist in the original dataset can be used.
-            shuffle(:obj:`bool`, default to `False`): Shuffle the dataset order when loading. Recommended True for training, False for
+            batch_size (`int`):
+                Size of batches to load from the dataset.
+            columns (`List[str]` or `str`, *optional*):
+                Dataset column(s) to load in the `tf.data.Dataset`. Column
+                names that are created by the `collate_fn` and that do not exist in the original dataset can be used.
+            shuffle(`bool`, defaults to `False`):
+                Shuffle the dataset order when loading. Recommended `True` for training, `False` for
                 validation/evaluation.
-            drop_remainder(:obj:`bool`, default ``False``): Drop the last incomplete batch when loading. Ensures
+            drop_remainder(`bool`, defaults to `False`):
+                Drop the last incomplete batch when loading. Ensures
                 that all batches yielded by the dataset will have the same length on the batch dimension.
-            collate_fn(:obj:`Callable`, optional): A function or callable object (such as a `DataCollator`) that will collate
+            collate_fn(`Callable`, *optional*):
+                A function or callable object (such as a `DataCollator`) that will collate
                 lists of samples into a batch.
-            collate_fn_args (:obj:`Dict`, optional): An optional `dict` of keyword arguments to be passed to the
+            collate_fn_args (`Dict`, *optional*):
+                An optional `dict` of keyword arguments to be passed to the
                 `collate_fn`.
-            label_cols (:obj:`List[str]` or :obj:`str`, default ``None``): Dataset column(s) to load as
+            label_cols (`List[str]` or `str`, defaults to `None`):
+                Dataset column(s) to load as
                 labels. Note that many models compute loss internally rather than letting Keras do it, in which case
```

Comment:
```suggestion
                Dataset column(s) to load as labels.
                Note that many models compute loss internally rather than letting Keras do it, in which case
```